core/src/main/java/org/geysermc/geyser/pack/GeyserResourcePackManifest.java

```diff
@@ -27,16 +27,22 @@
 
 import org.geysermc.geyser.api.packs.ResourcePackManifest;
 
+import java.util.Collection;
 import java.util.UUID;
 
-public record GeyserResourcePackManifest(Header header) implements ResourcePackManifest {
+    public record GeyserResourcePackManifest(Integer formatVersion, Header header, Collection<Module> modules, Collection<Dependency> dependencies) implements ResourcePackManifest {
 
-    public record Header(UUID uuid, int[] version) implements ResourcePackManifest.Header {
+        public record Header(UUID uuid, int[] version, String name, String description, String versionString, int[] minimumSupportedMinecraftVersion) implements ResourcePackManifest.Header {
```

Comment:
Remove the `versionString` member from
the Header record

Reply:
aka just leave the override?

Reply:
Yeah, versionString() is just a utility method, it doesn't exist in the manifest